presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetPageSourceFactory.java

```diff
@@ -205,6 +211,9 @@ public static ParquetPageSource createParquetPageSource(
             if (e instanceof PrestoException) {
                 throw (PrestoException) e;
             }
+            if (nullToEmpty(e.getMessage()).trim().startsWith("Corrupted statistics for column")) {
```

Comment:
Instead of relying on the error message, can't we just do:

```
if (e instanceof ParquetCorruptionException)
    throw new PrestoException(HIVE_BAD_DATA, e);
}
```